src/node.rs

```diff
@@ -582,6 +597,11 @@ impl<S: std::fmt::Debug + ForkSource> InMemoryNodeInner<S> {
                     }
                 };
 
+                log::trace!("Final gas estimation values");
+                log::trace!("tx_body_gas_limit: {}", tx_body_gas_limit);
+                log::trace!("gas_for_bytecodes_pubdata: {}", gas_for_bytecodes_pubdata);
+                log::trace!("overhead: {}", overhead);
+                log::trace!("full_gas_limit: {}", full_gas_limit);
```

Comment:
Used alongside logs above:

```
log::trace!("Gas Estimation Results:");
log::trace!("  tx_body_gas_limit: {}", tx_body_gas_limit);
log::trace!("  gas_for_bytecodes_pubdata: {}", gas_for_bytecodes_pubdata);
log::trace!("  overhead: {}", overhead);
log::trace!("  full_gas_limit: {}", full_gas_limit);
```

Logs are grouped by category (Gas Estimation Values and Gas Estimation Results), making it clear what each group represents. More of a suggestion.